unit-tests/run-unit-tests.py

```diff
@@ -132,11 +132,14 @@ def subprocess_run(cmd, stdout = None):
         if stdout  and  stdout != subprocess.PIPE:
             handle = open( stdout, "w" )
             stdout = handle
+        start_time = time.time()
         rv = subprocess.run( cmd,
                              stdout = stdout,
                              stderr = subprocess.STDOUT,
                              universal_newlines = True,
+                             timeout=200,
                              check = True)
+        log.d("Test took", time.time() - start_time, "seconds")

```

Comment:
make lowercase (I like debug messages to be lower-case)...
but note the indentation -- you wanted it to be indented?

Reply:
How do we know if it timed out? Does it throw?
Do we want it to throw? I certainly would want an error to be printed on timeout...

Reply:
It throws, I forgot to add the except. The indent is intentional, should I log after the unindent instead?

Reply:
No, it's fine just wanted you to be aware of it